cuda_bindings/docs/source/release/13.X.Y-notes.rst

```diff
@@ -12,7 +12,16 @@ Released on TBD
 Highlights
 ----------
 
+* Migrated wheel dependencies from individual NVIDIA packages to the ``cuda-toolkit`` metapackage for improved dependency resolution and version constraints.
 * Automatic CUDA library path detection based on ``CUDA_HOME``, eliminating the need to manually set ``LIBRARY_PATH`` environment variables for installation.
+* Removed NVCC compiler from ``[all]`` optional dependencies as it was previously included only to access NVVM library, which now has its own dedicated wheel.
+
+
+Dependency Changes
+------------------
+
+* The ``[all]`` optional dependencies now use ``cuda-toolkit`` with appropriate extras instead of individual packages.
+* **Breaking change**: NVCC compiler is no longer automatically installed with ``pip install cuda-python[all]``. Users who need the NVCC compiler should explicitly install it with ``pip install nvidia-cuda-nvcc-cu13`` or the appropriate version for their needs.
```

Comment:
As mentioned earlier this is not considered a breaking change. Please remove the emphasis. The remaining of the text is good and can be combined with the one from the previous section.